packages-ts/starknet-gauntlet/src/commands/base/executeCommand.ts

```diff
@@ -95,7 +95,7 @@ export const makeExecuteCommand = <UI, CI>(config: ExecuteCommandConfig<UI, CI>)
         id: makeCommandId(config.category, config.action, config.suffixes),
         contractAddress: c.contractAddress,
         flags: flags,
-        contract: new Contract(c.contract.abi, c.contractAddress, c.provider.provider),
+        contract: new Contract(c.contract.abi, c.contractAddress || '', c.provider.provider),
```

Comment:
Should revert once https://github.com/0xs34n/starknet.js/commit/be50c90ee6af29ab0ceddb679b5d33d5193b9a14 is in a release